src/main/java/carpetaddonsnotfound/lists/BlockList.java

```diff
@@ -136,6 +149,9 @@ public class BlockList {
           Blocks.CRIMSON_SLAB,
           Blocks.WARPED_SLAB,
           Blocks.MANGROVE_SLAB,
+          Blocks.CHERRY_SLAB,
+          Blocks.BAMBOO_SLAB,
+          Blocks.BAMBOO_MOSAIC_SLAB,
```

Comment:
Are you sure there isn't Bamboo Mosaic Block?

Reply:
Yes. Check line 132 where you will find `Bamboo_Mosaic`.

Reply:
yep sorry about that just making sure :)